pkg/supportbundle/defaultspec/spec.yaml

```diff
@@ -6,8 +6,6 @@ spec:
   collectors:
     - clusterInfo: {}
     - clusterResources: {}
-    - ceph: {}
-    - longhorn: {}
```

Comment:
Copying feedback from @divolgin here.  We likely do not want to remove stuff from the default spec because it is used when running:
```
kubectl support-bundle https://kots.io/
```

Reply:
I added two functions to remove the kURL collectors and analyzers instead, that way the default spec can remain unchanged.